historical_system_profiles/views/v0.py

```diff
@@ -49,18 +40,51 @@ def _get_current_names_for_profiles(hsps):
     return enriched_hsps
 
 
+def _filter_old_hsps(hsps):
+    """
+    removes any HSPs with a captured_date older than now minus the valid profile age
+    """
+    now = datetime.datetime.now(datetime.timezone.utc)
```

Comment:
Will this be based on the backends current time or the users current time? I would hope this doesnt cause a misunderstanding but I did wonder if we should standardize on midnight EST, for example, to make it clear to users.

Reply:
It would be 100% backend. Basically records would "fall off" and become invisible as the window moved forward.

Reply:
Users in different time zones may not understand the window if we advertise it as 7 days but ew don't advertise what the timezone is.

Reply:
per Jerome, having the records become unavailable after exactly 7 days (168 hours) is OK. For example, if your captured_date is Sunday at 5pm on your local time, the record will become unavailable next Sunday at 5:01pm your local time. If you had a DST event, this may shift by an hour in your local time, but would still be 168 hours if you measured via stopwatch.